### DISCUSSION TITLE
PAG文件耗电情况

### DISCUSSION THREAD

**Discussion by RayLiuLei** [Tencent/libpag#2221] [General] [answered]
Android上使用SDK，同样的场景下（UI场景下展示动效），PagView更耗电还是PagImageView更耗电

**Comment by kevingpqi123** [comment, marked as answer]
关注这个没啥意义吧，更应该关注的是性能，可以看下这篇文章：https://pag.art/docs/use-pagimageview.html
了解下 PAGView 和 PAGImageView 的实现机制就会有了答案
